components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -1078,9 +1085,101 @@ private String populateOauthParameters(OAuth2Parameters params, OAuthMessage oAu
         if (StringUtils.isNotBlank(oauthRequest.getParam(CLAIMS))) {
             params.setEssentialClaims(oauthRequest.getParam(CLAIMS));
         }
+        try {
+            handleOIDCRequestObject(oauthRequest, params);
+        } catch (RequestObjectException e) {
+            // All the error logs are specified at the time when throw the exception.
+            return EndpointUtil.getErrorPageURL(e.getErrorCode(), e.getErrorMessage(), null);
+        }
         return null;
     }
 
+    private void handleOIDCRequestObject(OAuthAuthzRequest oauthRequest, OAuth2Parameters parameters)
+            throws RequestObjectException {
+        validateRequestObject(oauthRequest);
```

Comment:
I think we are adding a new line after the method signature. It is in the new WSO2 code template file.